src/shared/lib/StoreUtils.ts

```diff
@@ -287,15 +295,20 @@ export async function fetchCnaOncoKbData(sampleIdToTumorType:{[sampleId: string]
                                          discreteCNAData:MobxPromise<DiscreteCopyNumberData[]>,
                                          client: OncoKbAPI = oncokbClient)
 {
-    if (discreteCNAData.result && discreteCNAData.result.length > 0) {
+    if (!discreteCNAData.result || discreteCNAData.result.length === 0) {
```

Comment:
This one fetches OncoKB data for the CNA table. We are getting `gene` and `sample id` information from the CNA data. Rest of the business logic is identical to the method where we get OncoKB data for mutation table.